### PR TITLE
Network documentation 2.0 (In progress)

### PR DESCRIPTION
**Pull request by engiValk**
-Files changed.
Comments for ClientHandler.java Finished.
Comments for MetricRecordingHandler.java Finished

Signed off.

engiValk.

### Contains

Just contains networking comments and javadocs.

### How to test

No testing.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: engine/src/main/java/org/terasology/network/internal/MetricRecordingHandler.java [hunk 1]
```diff
@@ -45,6 +51,12 @@ public void messageReceived(ChannelHandlerContext ctx, MessageEvent e) throws Ex
         ctx.sendUpstream(e);
     }
 
+    /**
+     * A metric function, increments AtmoicIntegers to keep track of data received, then pushes them downstream to be written.
```
Comment (pollend) [inline engine/src/main/java/org/terasology/network/internal/MetricRecordingHandler.java]: this is a comma splice can you reword this better.

File: engine/src/main/java/org/terasology/network/internal/ClientHandler.java [hunk 2]
```diff
@@ -66,6 +77,10 @@ public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e) {
     }
 
 
+    /**
+     *  Marks the joining of the server to be complete and sets the server in networkSystem.
```
Comment (pollend) [inline engine/src/main/java/org/terasology/network/internal/ClientHandler.java]: you have an extra space here before you're message.

Comment (engiValk) [reply engine/src/main/java/org/terasology/network/internal/ClientHandler.java]: I go back through files as I write these and update them all again once I've physically seen them being called. Most of these are placeholders while I keep working so I know where to go back and fix later.

File: engine/src/main/java/org/terasology/network/internal/ClientHandler.java [hunk 3]
```diff
@@ -53,6 +59,11 @@ public void channelDisconnected(ChannelHandlerContext ctx, ChannelStateEvent e)
         }
     }
 
+    /**
+     * Gets the message from the message event. Then queues it to the server for processing.
+     * @param ctx Channel Handler Context
+     * @param e
```
Comment (bpas247) [inline engine/src/main/java/org/terasology/network/internal/ClientHandler.java]: `e` needs a description

File: engine/src/main/java/org/terasology/network/internal/ClientHandler.java [hunk 4]
```diff
@@ -45,6 +45,12 @@ public ClientHandler(NetworkSystemImpl networkSystem) {
         this.networkSystem = networkSystem;
     }
 
+    /**
+     * Displays the "Disconnected from server message."
+     * @param ctx Channel Handler Context
+     * @param e
```
Comment (bpas247) [inline engine/src/main/java/org/terasology/network/internal/ClientHandler.java]: `e` needs a description here.

**Comment by jellysnake** [comment]
@engiValk
Is there any further movement on this?

**Comment by pollend** [comment]
closing this PR due to lack of response.

**Comment by jellysnake** [comment]
To add to Pollends comment, if you at any stage wish to continue working on this just reopen the PR and we can work out the best way to continue these changes.  
:)  
Documentation is _always_ welcome.

**Pull request closed without merging**